sdk/test-utils/recorder/src/utils/index.ts

```diff
@@ -533,7 +533,8 @@ export function maskAccessTokenInBrowserRecording(fixtures: string): string {
  * Sanitizes the scope url in the request bodies [Meant for cleaning the false positives in cred-scan reports]
  */
 export function sanitizeScopeUrl(body: string) {
-  return body.replace(/scope=https%3A%2F%2F[^&"]*/g, "scope=https%3A%2F%2Fsanitized%2F");
+  // TODO: We removed this implementation because it was too eager in removing extra scopes.
```

Comment:
can we remove this function? I feel like having a method called `sanitizeScopeUrl` gives one a false sense of security if one is not looking into the implementation.

Reply:
CC @HarshaNalluru

Reply:
@jeremymeng thoughts?

Reply:
Harsha and Will have talked about this. I don’t remember what was going to be the solution though, @HarshaNalluru , @willmtemple

Reply:
We could use a better name something like `replaceAADScopeValueWithPlaceHolder()`? 

@witemple-msft we want the recordings to only have the placeholder for the scope value.  What didn't work?  Could we fix by improving the regex?  Or does identity playback tests depend on the actual scope value?

Reply:
My current thoughts are to investigate and improve it with a better regex after this PR.

Jeremy's suggested name sounds better.

Reply:
@jeremymeng Scopes are part of the request, so if we change them, we have to be able to change them in a way that's consistent with the input that we provide to the client. For identity, we do a lot of testing with passing in scopes manually. If we are going to sanitize them, we need some way to get the "sanitized" value to pass in during playback so that the request will match.

Reply:
I have removed this function entirely. We may want to add it back, but for now i've just removed it.

Reply:
@HarshaNalluru FYI I restored this function in one of the latest commits.